Documentation/BasicUsage.md

```diff
@@ -0,0 +1,267 @@
+# Basic Usage
+
+For starters we're going to import everything ready to use.  
+Most of the included functionality is similar to how you would deal with WebRTC in your browser.  
+We support most of the official WebRTC APIs, see this [document](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection) for more details.
+
+```javascript
+import {
+	ScreenCapturePickerView,
+	RTCPeerConnection,
+	RTCIceCandidate,
+	RTCSessionDescription,
+	RTCView,
+	MediaStream,
+	MediaStreamTrack,
+	mediaDevices,
+	permissions,
+	registerGlobals
+} from 'react-native-webrtc';
+```
+
+## Defining Media Constraints
+
+```javascript
+var mediaConstraints = {
+	audio: {
+		googEchoCancellation: true,
```

Comment:
All of those have been dropped from WebRTC, let's just say `audio: true` here.

Reply:
Done in the next commit comming.